News-Android-App/src/main/res/xml/pref_general.xml

```diff
@@ -84,6 +83,21 @@
             android:title="@string/pref_title_OpenInBrowserDirectly"
             app:iconSpaceReserved="false"/>
 
+        <ListPreference
+            android:defaultValue="1"
+            android:entries="@array/pref_general_swipe_action"
+            android:entryValues="@array/pref_general_swipe_action_values"
+            android:key="sp_swipe_right_action"
+            android:title="Action when swiping right"
```

Comment:
The last thing is to convert both titles for the new settings to `@string` objects